aws/container-linux/kubernetes/ami.tf

```diff
@@ -14,6 +14,6 @@ data "aws_ami" "coreos" {
 
   filter {
     name   = "name"
-    values = ["CoreOS-${var.os_channel}-*"]
+    values = ["CoreOS-${var.os_channel}-${var.os_version}*"]
```

Comment:
this change has nothing to do with cloud provider, no?

Reply:
no, it doesn't